### DISCUSSION TITLE
❓ [QUESTION] Colab tutorial

### DISCUSSION THREAD

**Discussion by knc6** [mir-group/nequip#344] [Q&A] [answered]
Thanks for developing nequip and sharing the colab tutorial, which is very helpful.

The notebook mentions "We see that the model has converged to an energy accuarcy < 1meV/atom and a force accuracy of approx. 40 meV/A within 5 minutes and trained on only 100 samples." 

1) Its unclear to me which number is referred to in this statement. Is it the e_mae e/N_mae?

![image](https://github.com/mir-group/nequip/assets/16902896/424416e8-0ce0-4fce-a1ef-e762f0ab5f6d)

2) Is there any config file to reproduce the publication results from the original nequip paper? Perhaps this yaml [file](https://github.com/mir-group/nequip/blob/main/configs/full.yaml)?

**Comment by Linux-cpp-lisp** [comment, marked as answer]
Hi @knc6 ,

Thanks for your interest in our code!

 1.  `e/N_mae` is the per-atom error (i.e. energy units/atom). Quoting the notebook:
       > The units of the reference data are in kcal/mol and A. If you're more familiar with eV, remember 1 kcal/mol is chemical accuracy and is approximately 43 meV
       
       You can see that 0.0225 kcal/mol * 43 meV / (kcal/mol) is about 1 meV.
 2. See https://github.com/mir-group/nequip-input-files.  But as that README says, new applications (and not exact reproducing) should use newer versions of the code and take `configs/example.yaml` as a starting point.
